api/tests/utils/fixtures_tests.py

```diff
@@ -33,3 +49,59 @@ def config_rollback():
 def app_config():
     app_config = create_app_config()
     return app_config
+
+
+@pytest.fixture
+def camera_sample():
+    camera_sample = copy.deepcopy(example)
+    return camera_sample
+
+
+@pytest.fixture
+def rollback_screenshot_camera_folder():
+    yield None
```

Comment:
What is the purpose of this?

Reply:
The point is that, when we create a camera to test something, that camera creates a folder within the path "data / processor / static / screenshots / <camera_id>". So, once the test was done, we had to delete said directory and all its contents. This task is taken by this fixture and its tearDown code.

Reply:
I mean the `yield None`

Reply:
Pytest provide an intuitive way to run setUp and tearDown code with fixtures.
Simply put, the setup code runs before the return statement. But if we want to add tearDown code, we just have to change return -> yield, and write our tearDown code after the yield statement.
It is explained here: https://docs.pytest.org/en/stable/fixture.html#yield-fixtures-recommended